bot/bot.py

```diff
@@ -36,6 +41,16 @@ def __init__(self, redis_session: RedisSession, **kwargs):
         self.redis_session = redis_session
         self.loop.create_task(self.check_channels())
         self.loop.create_task(self.send_log(self.name, "Connected!"))
+        self.loop.create_task(self.process_ext_not_found())
+
+    async def get_unloaded_extensions(self) -> None:
+        """Get all the unloaded extensions from cache."""
+        # For testing sir-lancebot#705
+        await self.unloads_cache.set("bot.exts.evergreen.catify", "https://dummy-msg-link.com")
+        await self.unloads_cache.set("bot.exts.evergreen.xkd", "https://dummy-msg-link.com")
```

Comment:
Are these for local testing? They should not be left in production.

Reply:
Yeah these are for local testing, I have hence left a comment over that and in the PR description. I will remove that before it gets merged.